contracts/src/v0.8/functions/dev/1_0_0/FunctionsRouter.sol

```diff
@@ -318,4 +333,11 @@ contract FunctionsRouter is RouterBase, IFunctionsRouter, FunctionsSubscriptions
     _validateOwnership();
     _;
   }
+
+  modifier onlySenderThatAcceptedToS() override {
```

Comment:
Do we validate this when sending requests too? Or just relying on existing subscriptions? If that's the case then how do we deal with blocked users who had subscriptions before?

Reply:
No, only subscription management. There is no limiting of requests.

That's a layer I haven't thought about. Currently blocked users just cannot manage subscriptions, but maybe we should also close down any subscriptions that they are currently the owner of. Let's talk to product about that.

Reply:
Yeah, let's follow up separately.